### PR TITLE
Improve Workflow

### PR DESCRIPTION
**Pull request by CristianUser**
### Stories?
https://trello.com/c/50UjFJM0/5-move-some-env-var-to-setup-object
https://trello.com/c/1jMIuHmL/9-setup-github

## What does this PR do?
Adds setup object to init function
Adds .github folder

### Why are we doing this? Any context or related work?
To have more control in the development process

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,9 +25,9 @@
     "eslint": "^5.16.0",
     "eslint-config-prettier": "^6.9.0",
     "eslint-plugin-prettier": "^3.1.2",
-    "husky": "^3.1.0",
+    "husky": "^4.2.3",
```
Comment (oliverfb) [inline package.json:28]: You don't have to do this now but you should think about pinning the versions for your packages at some point. More info:
https://docs.renovatebot.com/dependency-pinning/

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm i
+    - run: npm run lint
```
Comment (oliverfb) [inline .github/workflows/lint.yml]: Consider also `rm -rf` node_modules and any build files!

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+[Issue/Story](LINK_TO_STORY)
+
+### Why are we doing this? Any context or related work?
+
+#### Where should a reviewer start?
+
+#### Related PR
+
+### Manual testing steps?
```
Comment (oliverfb) [inline .github/pull_request_template.md]: Maybe don't ask for it as a question? 😄

File: package.json (hunk 1 above)
Comment (CristianUser) [reply package.json:28]: Yes, but this was added by a command to setup `pre-commit`, and I wasn't using it before

File: .github/workflows/lint.yml (hunk 2 above)
Comment (CristianUser) [reply .github/workflows/lint.yml]: Github removes the complete environment by default

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+### What does this PR do?
+
+[Issue/Story](LINK_TO_STORY)
+
+### Why are we doing this? Any context or related work?
+
+### Where should a reviewer start
```
Comment (oliverfb) [inline .github/pull_request_template.md:7]: Only manual testing steps did not need to be a question, this one does need to be

**Pull request merged**